Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -78,195 +81,234 @@ Copyright Notice
 
-   o  Scalability issues
+   The infrastructure on which the experiments have been run has also
+   involved significant effort to build, principally in its operational
+   aspects: the coordination of a large pool of volunteers in the
+   implementation of the testbed is itself an exercise with relevance to
+   the real world, and commentary on the corresponding logistics is also
+   included in this document.
 
-   o  Multiple zone file signers
+2.1.  Yeti-Root Zone Distribution
 
-   Starting from May 2015, three coordinators began to build this live
-   experimental environment and called for participants.  At the time of
-   writing, there are 25 Yeti root servers with 15 operators, and
-   experimental traffic from volunteers, universities, DNS vendors,
-   mirrored traffic, non-Yeti traffic, and RIPE Atlas probes.
+   o  What are the scaling properties of Yeti-Root zone distribution as
+      the number of Yeti-Root servers, Yeti-Root server instances or
+      intermediate distribution points increase?
 
-   Note that the Yeti DNS Project has complete fealty to IANA as the DNS
-   name space manager.  All IANA top-level domain names will be
-   precisely expressed in the Yeti DNS system, including all TLD data
-   and meta-data[Root-Zone-Database].  Therefore, the Yeti DNS Project
-   is never an "alternative root" in the usual sense of that term.  It
-   is expected to inform the IANA community by peer-reviewed science as
-   to future possibilities to consider for the IANA root DNS system.
+2.2.  Yeti-Root Server Names and Addressing
 
+   o  What naming schemes other than those closely analogous to the use
+      of ROOT-SERVERS.NET in the production root zone are practical, and
+      what are their respective advantages and disadvantages?
 
+   o  What are the risks and benefits of signing the zone that contains
+      the names of the Yeti-Root servers?
 
+   o  What automatic mechanisms might be useful to improve the rate at
+      which clients of Yeti-Root servers are able to react to a Yeti-
+      Root server renumbering event?
 
 
-Song, et al.            Expires December 11, 2017               [Page 3]
+
+
+Song, et al.            Expires December 18, 2017               [Page 4]
 
 Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
 
 
-   In order to let people know the technical activities in Yeti DNS
-   Project, this document reports and discusses issues on root DNS
-   services, based on experiences so far from the testbed construction
-   and experiments in the Yeti DNS Project.
+2.3.  IPv6-Only Yeti-Root Servers
+
+   o  Are there negative operational effects in the use of IPv6-only
+      Yeti-Root servers, compared to the use of servers that are dual-
+      stack?
+
+   o  What effect does the IPv6 fragmentation model have on the
+      operation of Yeti-Root servers, compared with that of IPv4?
+
+2.4.  DNSSEC in the Yeti-Root Zone
```

Comment:
In this part , Yeti has findings in two aspects: 1) Yeti KSK rollover introduces large response packets(with 5~6 keys up to 1900+) by querying DNSKEY rrset +dnssec.  Timeout is spotted in udp with the help of atlas monitroing. 2) It is reported in Yeti blog that BIND9 has a issue when new views are created during the rolling which has been introduced in the 03-version. draft.